cf_xarray/utils.py

```diff
@@ -57,3 +61,57 @@ def invert_mappings(*mappings):
 
 def always_iterable(obj: Any) -> Iterable:
     return [obj] if not isinstance(obj, (tuple, list, set, dict)) else obj
+
+
+def parse_cf_table(uri: Union[str, Path], verbose=False):
```

Comment:
Should we parse this once at release-time and bundle a pickled dictionary? Or write a valid python file that creates the dictionary?